examples/using-loadable-components/README.md

```diff
@@ -0,0 +1,87 @@
+# Gatsby + loadable-components
+
+This repo is meant to illustrate the pattern of adding loadable-components to a Gatsby project. In this case, the Header component is being code-split out while mainting SSR.
+
+```jsx
+// layout.js
+
+const Header = loadable(() => import("./header"))
+
+<Header siteTitle="My Title" />
+```
+
+**For a full breakdown of this pattern, [check out this blog post]().**
```

Comment:
Is there already a slug/URL defined? It can be prefilled

Reply:
Just removing this line for now until the slug is ready.